app/uk/gov/hmrc/agentservicesaccount/views/pages/agent_services_account.scala.html

```diff
@@ -22,20 +22,18 @@
 
 @analyticsAdditionalJs = {
 
-    @featureSwitch("features.showAgentInvitationsLink", isWhitelisted) {
+
```

Comment:
can this featureSwitch function be removed if all uses of it have been removed

Reply:
We need the function for the remaining flag